src/core/utils/heights.ts

```diff
@@ -7,12 +7,12 @@ export const HEADER_HEIGHT = 64;
 export const HEADER_BOTTOM_MARGIN = 24;
 
 /**
- * Calculates the maximum height for a component by subtracting the total of given heights from 100vh.
+ * Calculates the maximum height for a component by subtracting the total of given heights from 100dvh.
  *
  * @param {...number} heights - An array of heights in pixels.
  * @returns {string} The CSS calc expression for the maximum height.
  */
 export const componentMaxHeight = (...heights: number[]): string => {
   const totalHeight = heights.reduce((sum, height) => sum + height, 0) + "px";
-  return `calc(100vh - ${totalHeight})`;
+  return `calc(100dvh - ${totalHeight})`;
```

Comment:
@aralovelace this was the iOS fix. [dvh](https://developer.mozilla.org/en-US/docs/Web/CSS/length#dynamic_viewport_units) is the "dynamic viewport height" that takes into account other regions like the iOS address bar. [It's pretty new](https://caniuse.com/?search=dvh) but there are no Firefox users on iOS (it's all Webkit)

Reply:
TIL 🧠